Sources/Views/MessageInputBar.swift

```diff
@@ -573,14 +575,18 @@ open class MessageInputBar: UIView {
     ///   - animated: If the layout should be animated
     open func setStackViewItems(_ items: [InputBarButtonItem], forStack position: InputStackView.Position, animated: Bool) {
         
+        func setupStackViewItem(_ item: InputBarButtonItem) {
```

Comment:
Do you mind if we revert this change? I'd rather not add it if it only trims a single line

Reply:
No, I don't mind.